tests/dummy/app/helpers/eq.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export function eq(params) {
```

Comment:
If we decide that we really do need truth helpers in handlebars templates, this library may be of interest:
https://www.npmjs.com/package/ember-truth-helpers